### PR TITLE
api v1 - restapi renamed to apipie

### PR DESCRIPTION
**Pull request by mbacovsky**

### CODE REVIEW

File: Rakefile [hunk 1]
```diff
@@ -2,9 +2,6 @@ require File.expand_path('../config/application', __FILE__)
 require 'rake'
 include Rake::DSL
 
-if Rails.env.test?
-  require 'single_test'
-  SingleTest.load_tasks
-end
+SingleTest.load_tasks if defined? SingleTest
```
Comment (ohadlevy) [inline Rakefile:5]: why even load this if we are not in testing?

File: bundler.d/test.rb [hunk 2]
```diff
@@ -5,4 +5,5 @@
   gem 'rake'
   gem 'redcarpet'
   gem 'single_test'
+  gem 'pry'
 end
```
Comment (ohadlevy) [inline bundler.d/test.rb]: not sure if it make sense, (e.g. for CI)

File: Rakefile (hunk 1 above)
Comment (mbacovsky) [inline Rakefile:5]: we load the tasks it if the SingleTest is available and that happens only in test env

File: bundler.d/test.rb (hunk 2 above)
Comment (mbacovsky) [inline bundler.d/test.rb]: I used pry to debug a failing test, so it makes sense for me to have in the testing env, but no problem to remove it

Comment (mbacovsky) [inline bundler.d/test.rb]: Removed pry from the test.rb

**Comment by ohadlevy** [comment]
merged!

**Pull request closed without merging**